### PR TITLE
Adjust selection range for wrap element when empty selection is in start or end tag.

### PR DESCRIPTION
**Pull request by angelozerr**
Adjust selection range for wrap element when empty selection is in start or end tag.

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
Here a demo:

SurroundWithTagsAdjust

File: org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/commands/SurroundWithCommand.java [hunk 1]
```diff
@@ -138,9 +141,6 @@ protected Object executeCommand(DOMDocument document, ExecuteCommandParams param
 					startText.append(tag);
 				}
 				startText.append(">");
-				if (emptySelection && snippetsSupported) {
```
Comment (datho7561) [inline org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/commands/SurroundWithCommand.java:141]: Do you think this feature isn't helpful? If so, why?

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/commands/SurroundWithCommand.java (hunk 1 above)
Comment (angelozerr) [reply org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/commands/SurroundWithCommand.java:141]: Ok I support again this pattern

**Review by datho7561 — APPROVED** [review]
Looks good! Thanks Angelo!

**Pull request merged**